zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -387,6 +421,25 @@ private static List<String> applyProcessInstanceSortField(
     return validationErrors;
   }
 
+  private static List<String> applyProcessDefinitionSortField(
+      final String field, final ProcessDefinitionSort.Builder builder) {
+    final List<String> validationErrors = new ArrayList<>();
+    if (field == null) {
+      validationErrors.add(ERROR_SORT_FIELD_MUST_NOT_BE_NULL);
+    } else {
+      switch (field) {
+        case "processDefinitionKey" -> builder.processDefinitionKey();
```

Comment:
I see we have over the sort builder the formKey, are we planning add here as well?

Reply:
Same as above about formKey.